src/main/resources/templates/omissions.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  layout:decorate="~{layout}">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Omissions</title>
+</head>
+
+<body>
+  <div layout:fragment="content">
+
+    <h1 class="text-green-200 font-sarabun font-light font text-32">
+      Omissions <i id="omissionsInfo" class="fa-solid fa-circle-question text-black"></i>
```

Comment:
empty italics tag?

Reply:
The `i` tag originally meant "italics," but has been expanded. It's the tag Font Awesome recommends using for icons, and the classes cause Font Awesome icons to be added (in this case, a solid circle with question mark)